### PR TITLE
chore: rename "project delete" to "app delete"

### PR DESCRIPTION
**Pull request by efekarakus**
*By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.*

### CODE REVIEW

File: internal/pkg/cli/app_delete_test.go [hunk 1]
```diff
@@ -0,0 +1,328 @@
...
+					mocks.envDeleter.EXPECT().Execute().Return(nil),
+
+					// emptyS3bucket
+					mocks.store.EXPECT().GetApplication(mockAppName).Return(mockApp, nil),
+					mocks.deployer.EXPECT().GetRegionalAppResources(mockApp).Return(mockResources, nil),
+					mocks.spinner.EXPECT().Start(deleteAppCleanResourcesStartMsg),
+					mocks.bucketEmptier.EXPECT().EmptyBucket(mockResources[0].S3Bucket).Return(nil),
+					mocks.spinner.EXPECT().Stop(log.Ssuccess(deleteAppCleanResourcesStopMsg)),
+
+					// deleteProjectPipline
```
Comment (bvtujo) [inline internal/pkg/cli/app_delete_test.go:232]: nit: deleteAppPipeline
```suggestion
					// deleteApplicationPipline
```

File: internal/pkg/cli/app_delete.go [hunk 2]
```diff
@@ -0,0 +1,334 @@
...
+	if err != nil {
+		return fmt.Errorf("confirm app deletion: %w", err)
+	}
+	if !manualConfirm {
+		return errOperationCancelled
+	}
+	return nil
+}
+
+// Execute deletes the project.
```
Comment (bvtujo) [inline internal/pkg/cli/app_delete.go:156]:
```suggestion
// Execute deletes the application.
```

File: internal/pkg/cli/app_delete_test.go [hunk 3]
```diff
@@ -0,0 +1,328 @@
...
```
Comment (bvtujo) [inline internal/pkg/cli/app_delete_test.go:191]:
```suggestion
					// deleteApplicationPipline
```

File: internal/pkg/cli/app_delete_test.go (hunk 1 above)
Comment (iamhopaul123) [reply internal/pkg/cli/app_delete_test.go:232]: should this be pipeline btw

Comment (efekarakus) [reply internal/pkg/cli/app_delete_test.go]: Done!

File: internal/pkg/cli/app_delete.go (hunk 2 above)
Comment (efekarakus) [reply internal/pkg/cli/app_delete.go]: Done!

File: internal/pkg/cli/app_delete_test.go (hunk 3 above)
Comment (efekarakus) [reply internal/pkg/cli/app_delete_test.go]: Done!

**Pull request merged**